php/blocks/sidebar/disk-info.php

```diff
@@ -0,0 +1,49 @@
+<div class="box disk-info">
+	<h3>Disk Info</h3>
+
+	<?php
+	$v_free_space_mb = round( disk_free_space( '/' ) / 1024 / 1024, 1 );
+	$v_free_space_gb = round( $v_free_space_mb / 1024, 1 );
+	$v_total_space   = round( disk_total_space( '/' ) / 1024 / 1024 / 1024, 0 );
+	$v_used_space    = round( $v_total_space - ( $v_free_space_mb / 1024 ), 1 );
+	$v_used_percent  = round( ( $v_used_space / $v_total_space ) * 100, 0 );
+	$v_low_space     = ( 200 > $v_free_space_mb );
+	$v_free_space    = ( $v_free_space_mb < 1024 ) ? $v_free_space_mb . ' MB' : $v_free_space_gb . ' GB';
+	?>
+	<div class="disk">
+		<div class="disk-header-meta">
+			<div class="disk-label">Virtual Machine</div>
```

Comment:
```suggestion
			<div class="disk-label">VVV Guest Virtual Machine</div>
```

Lets eliminate as much ambiguity as possible